<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRecurrence</class>
 <widget class="QWidget" name="FormRecurrence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string comment="GRP_RECURRENCE">Recurrence</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBox_Select">
          <item>
           <property name="text">
            <string>daily</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>weekly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>monthly</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_WeekDay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Monday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tuesday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Wednesday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Thursday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Friday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Saturday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sunday</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_MonthDay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>01.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>02.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>03.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>04.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>05.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>06.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>07.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>08.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>09.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>21.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>22.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>26.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>27.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>28.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>29.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>31.</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_Year">
          <item>
           <property name="text">
            <string>year</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Generate">
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_Dates">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>108</width>
          <height>0</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
